apph-front/src/utils/types/Table.ts

```diff
@@ -10,7 +10,7 @@ export interface ITable {
   url: string;
   tags: ITag[];
   format: string;
-  details: JSX.Element;
+  details?: JSX.Element;
 }
 
 export interface IPhotoDetails {
```

Comment:
```suggestion
export interface ITableDetails {
```
Mettez les noms d'interface en cohérent, si t'as changé `IPhoto` avec `ITable`